lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -321,8 +322,8 @@ function dosomething_campaign_preprocess_hot_module(&$vars, &$wrapper) {
     array('dosomethingCampaign' =>
       array(
         'goals' => array(
-          'dates' => json_encode($dates),
-          'quantities' => json_encode($quantities),
+          'dates' => (isset($dates)) ? json_encode($dates) : null,
```

Comment:
Actually, is the `json_encode` necessary here? I _think_ it might be safe to remove and then you wouldn't have to use `JSON.parse()` on the client-side.